jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb

```diff
@@ -527,6 +527,16 @@
     "    # Select only the items with selected observed date\n",
     "    month_items = items_gdf.loc[items_gdf['observed'] == observed]\n",
     "    \n",
+    "    # The heatmap rendering requires 2 or more distinct geometries. If there is only one,\n",
+    "    # let's duplicate it and offset the new geometry by a small amount\n",
+    "    if len(month_items) == 1:\n",
+    "        geom = month_items.iloc[0]['geometry']\n",
+    "        new_geom = shapely.affinity.translate(geom, xoff=0.01)\n",
```

Comment:
@BradNeuberg  need to add import shapely

Reply:
Thanks! Great point :/

Reply:
Actually, it's imported earlier in the notebook: `from shapely.geometry import shape, Polygon`

Reply:
The interesting thing is it worked fine without that import, due to some other things that were imported earlier, but for correctness I added `import shapely`.